tests/test_closeable.py

```diff
@@ -0,0 +1,38 @@
+'''
+Test creating for java.io.Closeable dunder
+'''
+
+from __future__ import absolute_import
+import unittest
+from jnius import autoclass, JavaException, protocol_map
+
+
+class TestConstructor(unittest.TestCase):
+    '''
+    TestCase for using java.io.Closeable dunder
+    '''
+
+    # this checkes that java.lang.AutoCloseable instances gain the correct dunder methods
```

Comment:
I believe these comments can be made into docstrings of the test. Anyway they should be inside the method, not before (same for next test).